crypto/threads_pthread.c

```diff
@@ -620,11 +670,39 @@ CRYPTO_RWLOCK *CRYPTO_THREAD_lock_new(void)
     return lock;
 }
 
+# define BT_BUF_SIZE 1024
```

Comment:
Should this be in a `#  ifdef REPORT_RWLOCK_CONTENTION`?

Reply:
It could be, but its not going to hurt anything as it exists now.  I can fix it up on merge